routes/homeRoutes.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const router = express.Router();
+const { getTweets } = require('../controllers/TweetController');
+
+
+router.get('/' , getTweets);
+
+
+
+module.exports = router;
```

Comment:
I do not think you need this file, you do not need to handle the base path if you do not have anything here

Reply:
ok sir